src/components/PublicRoute/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
This component and PrivateRoute are too similar. Can't we somehow reuse the code from both?